bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/editor/targetComponent/EdCompTargetList.java

```diff
@@ -590,23 +717,32 @@ public void actionPerformed(final ActionEvent e) {
                     env.setUserTargets(env.getUserTargets().append(newTarget));
             dataObject.setTargetEnvironment(newEnv);
         } else {
-            final GuideGroup group = TargetSelection.getGuideGroup(dataObject.getTargetEnvironment(), obsComponent);
-            if (group != null) {
-                final TargetEnvironment env = dataObject.getTargetEnvironment();
+            final Option<IndexedGuideGroup> iggOpt =
+                    TargetSelection.getIndexedGuideGroupForNode(dataObject.getTargetEnvironment(), obsComponent);
+            iggOpt.foreach(igg -> {
+                final GuideGroup origGroup    = igg.group();
+
+                // This used to be done with origGroup.cloneTargets(), but if origGroup is an auto group, this creates
+                // another auto group, which is problematic, so we specifically force a manual group to be created
+                // and receive an appropriate name.
+                final String newGroupName     = origGroup.getName().filter(s -> !s.isEmpty()).getOrElse("Manual Group");
+                final GuideGroup newGroup     = GuideGroup.create(newGroupName, origGroup.getAll()).cloneTargets();
+
+                final TargetEnvironment env   = dataObject.getTargetEnvironment();
```

Comment:
In `GuideGrp` there is a `toManualGroup` method.  We can add that to `GuideGroup` as well and then maybe just `cloneTargets().toManualGroup()`.